src/Socket_p.h

```diff
@@ -362,7 +359,7 @@ namespace Arcus
             return;
         }
 
-        uint32_t message_size = message->ByteSize();
```

Comment:
We are using Protobuf 3.0.0. The `ByteSize` function was deprecated in version 3.4, so this should not be a problem in our Protobuf version.